pxr/usd/usdShade/validators.cpp

```diff
@@ -192,13 +193,45 @@ _ShaderPropertyTypeConformance(const UsdPrim &usdPrim)
     return errors;
 }
 
+
+static
+UsdValidationErrorVector
+_MaterialBindingApiChecker(const UsdPrim &usdPrim)
+{
+    UsdValidationErrorVector errors;
+
+    if (!usdPrim.CanApplyAPI<UsdShadeMaterialBindingAPI>()){
```

Comment:
Question for the group: I know the current check doesn't check for it, but should the test flag a warning (not an error) when the MaterialBindingAPI is applied to a prim that it can't be applied to?

Reply:
What is an example of such an application @dgovil? I thought MaterialBindingAPI can be applied to any prim which wants to provide a material:binding relationship.

Reply:
Oh is MaterialBindingAPI unbounded? I thought it required at least being a gprim?

Reply:
I think a prim's ancestor also particate in determining what material is bound: https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usd/usdShade/materialBindingAPI.cpp#L718